charts/elastic-agent/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
```

Comment:
Perhaps we should update these notes with information on how to validate the agent is working?

Reply:
The agent seems to be a no-go for now because of their insistence on installing a service, regardless of what the user wants. I'm gonna pull this chart from the Helm packages for now until they fix the app.